gradle/libs.versions.toml

```diff
@@ -18,6 +18,7 @@ opentest4j = "1.3.0"
 openTestReporting = "0.1.0-M2"
 surefire = "3.5.0"
 xmlunit = "2.10.0"
+javadiffutils = "4.12"
```

Comment:
There's no need to extract the version into a variable if it's only referenced once. Please inline it.